src/main/java/org/catrobat/jira/timesheet/rest/TimesheetRest.java

```diff
@@ -829,7 +829,7 @@ public Response deleteTimesheetEntry(@Context HttpServletRequest request,
             state = Timesheet.State.ACTIVE;
         }
 
-        int deducted_hours = sheet.getHoursCompleted() - entry.getDurationMinutes() / 60;
+        int deducted_hours = sheet.calculateTotalHours() - entry.getDurationMinutes() / 60;
```

Comment:
Why are you not using the stored property from the database but re-calculating it on every call?